client/src/m_options.cpp

```diff
@@ -216,6 +226,11 @@ value_t OnOffAuto[3] = {
 	{ 2.0, "Auto" }
 };
 
+value_t DemoRestrictions[2] = {
```

Comment:
Whys is 0 allow and 1 restrict? This is OK as far as setting in the application. This is confusing in odamex.cfg because having cl_autorecord_filter_duel set to 1 would imply that it would record but it's the opposite.

To me it seems we should flip the DemoRestrictions values and default the cvar to 1.